charts/orchestrator/values.yaml

```diff
@@ -119,10 +119,10 @@ postgresql:
   enabled: true
   fullnameOverride: "postgresql"
   auth:
-    username: "postgres"
+    username: "testuser"
     password: "default"
     postgresPassword: "default"
```

Comment:
@adkumar1 I'd suggest to use existing postgresql secret with auto-generated password instead of Insecure password exposure

Reply:
Did the changes

Reply:
@adkumar1 I'm afraid that may not work, just removing the password possibly makes the helm chart unable to install since the app won't be able to login. I suggest to look at other apps which implemented the psql secret successfully i.e. see the linked PR in the following issue: https://github.com/eclipse-tractusx/puris/issues/260

Reply:
oh I think I got confused with empty existingSecret, but looks like in this case it gets created anyway with default 'postgres' secret name

Reply:
yes right, I did the testing post making above changes a well. I did deployed it & it work properly